xarray/core/dataarray.py

```diff
@@ -6352,7 +6352,7 @@ def curvefit(
         ...     param="time_constant"
         ... )  # doctest: +NUMBER
         <xarray.DataArray 'curvefit_coefficients' (x: 3)>
-        array([1.0569203, 1.7354963, 2.9421577])
+        array([1.05692036, 1.73549638, 2.94215771])
```

Comment:
Unfortunately I think this needs reverting again...

Reply:
I copied this from the error report from the GitHub workflow yesterday when it failed on the aarch64 string I put in; it seems to be passing with it as is.